rfc/004-feature-template.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: badges_node
+- Start Date: 2020-08-24
+- RFC PR: [amundsen-io/rfcs#4](https://github.com/amundsen-io/rfcs/pull/4)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000)
+
+# Separate badges from tags
+
+## Summary
+
+Create a new node type for neo4j which would represent Badges that is separate from the Tag node.
```

Comment:
Amundsen metadata has different backends (neo4j, atlas, gremlin is coming). I think `creating a separate entity for badge` would be more suitable for our case.

Reply:
also should we cover column level badge discussion here?